lib/matplotlib/bezier.py

```diff
@@ -239,15 +239,114 @@ def __init__(self, control_points):
         coeff = [math.factorial(self._N - 1)
                  // (math.factorial(i) * math.factorial(self._N - 1 - i))
                  for i in range(self._N)]
-        self._px = self._cpoints.T * coeff
+        self._px = (self._cpoints.T * coeff).T
 
     def __call__(self, t):
-        return self.point_at_t(t)
```

Comment:
Previous code could not accept vector input. Instead, first reshape `._orders` and `t` to allow them to broadcast, then use same formula as before.